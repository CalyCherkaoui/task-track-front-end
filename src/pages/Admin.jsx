import React from 'react';

const Admin = () => (
  <div>
    <h1>Admin Task & Track App</h1>
  </div>
);

export default Admin;
