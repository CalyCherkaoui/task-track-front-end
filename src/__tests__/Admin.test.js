import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../store';
import Admin from '../pages/Admin';

describe(
  'Renders correctly the Admin page',
  () => {
    it(
      'Matches Admin snapshot',
      () => {
        const rendered = renderer.create(
          <Provider store={store}>
            <Router>
              <Admin />
            </Router>
          </Provider>,
        );

        expect(rendered).toMatchSnapshot();
      },
    );
  },
);
