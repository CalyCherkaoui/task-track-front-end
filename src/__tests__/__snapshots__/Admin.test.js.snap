// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly the Admin page Matches Admin snapshot 1`] = `null`;
